src/components/Banner.jsx

```diff
@@ -21,18 +22,36 @@ const TitleBanner = styled.h1`
   margin-bottom: 1rem;
   text-decoration: underline;
 `;
-export default function Banner({ title, subtitle }) {
+
+const BackgroundBanner = styled.img`
+  height: 600px;
+  object-fit: cover;
+  box-sizing: border-box;
+  position: relative;
+  right: -30rem;
```

Comment:
What is the purpose of the right -30 rem ? Are you sure this is doing something ?

Reply:
It place the background's truck at the right of the banner